plugins/wfp/management/commands/wfp_etl_Under5.py

```diff
@@ -192,23 +192,21 @@ def run(self):
             logger.info(
                 f"---------------------------------------- Beneficiary N° {(index+1)} {instance['entity_id']}-----------------------------------"
             )
+            instance["journey"] = self.journeyMapper(instance["visits"])
             beneficiary = Beneficiary()
-            if instance["entity_id"] not in existing_beneficiaries:
+            if instance["entity_id"] not in existing_beneficiaries and len(instance["journey"][0]["visits"]) > 0:
                 beneficiary.gender = instance["gender"]
                 beneficiary.birth_date = instance["birth_date"]
                 beneficiary.entity_id = instance["entity_id"]
                 beneficiary.save()
                 logger.info(f"Created new beneficiary")
             else:
-                beneficiary = Beneficiary.objects.get(entity_id=instance["entity_id"])
-            instance["journey"] = self.journeyMapper(instance["visits"])
+                beneficiary = Beneficiary.objects.filter(entity_id=instance["entity_id"])
```

Comment:
Are sure this will work? This means `beneficiary` will be assigned a queryset instead of an object instance, no? Maybe `filter().first()`? Or catch the not found exception?

Reply:
you are right, it sould be filter().first(). Let me change it.